docs/content/docs/data-structures/tree.md

```diff
@@ -270,10 +267,21 @@ const containerSchema: ContainerSchema = {
 const { container, services } = await client.createContainer(containerSchema);
 ```
 
-Apply the schema to the tree by passing the tree configuration object to the `ITree.schematize()` method.
+Use `ITree.viewWith` to create a `TreeView` based on your tree configuration.
+Tree views provide schema-dependent APIs for viewing and editing tree data.
 
 ```typescript
-const stickyNotesTreeView = container.initialObjects.appData.schematize(appTreeConfiguration);
+const stickyNotesTreeView = container.initialObjects.appData.viewWith(appTreeConfiguration);
+```
+
+When the tree is first created, this schema along with some initial data can be applied to the tree using `TreeView.initialize`.
+Note that the parameter to initialize must conform to the root schema, `App`. So in this example, it has a single `items` property. The value of the `items` property specifies that the items array is empty. It is not a requirement that the initial tree be empty: you can assign one or more groups or notes to the initial tree.
+
+```typescript
+stickyNotesTreeView.initialize(
+    // This uses POJO (plain-old javascript object) mode, but `new App({ items: [] })` is also supported and equivalent.
```

Comment:
maybe overly detailed for the audience?

IMO if we're only going to include one, I'd kind of prefer the `new App` one. It's easier to understand and is more likely to put people in a pit of success (less compiler inference -> more likely the error message will be more intelligible)

Reply:
I think having both is fine but maybe in separate code blocks with some plain text explanation? The comment makes it a bit harder to read imo